src_cam/zm_microphysics.F90

```diff
@@ -627,6 +627,10 @@ subroutine zm_mphy(su,    qu,   mu,   du,   eu,    cmel,  cmei,  zf,   pm,   te,
 ! initialization
 !ccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
 
+  ! OSLO_AERO begin
+  call endrun('zm_microphysics not supported in oslo-aero')
+  ! OSLO_AERO end
+
```

Comment:
Two notes:
1. All the code in this subroutine below this `endrun` call cannot be called so removing it saves compile time.
2. ZM microphysics is going away in the next CAM update so this module can be removed (now or then).